.github/workflows/upgrade-e2e.yml

```
@@ -0,0 +1,41 @@
+---
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+    tags:
+      - 'v**'
+
+name: Upgrade tests
```

```suggestion
name: Upgrade Tests
```

(and maybe put the workflow name at the top of the file)